workspace/files/home/ubuntu/.bashrc

```diff
@@ -41,7 +41,7 @@ if [ -f /etc/bash_completion ] && ! shopt -oq posix; then
     . /etc/bash_completion
 fi
 
-PS1='\[\033[01;32m\]${C9_USER}\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]$(__git_ps1 " (%s)") $ '
+PS1='\[\033[01;32m\]${C9_USER}/${C9_PROJECT::16}\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]$(__git_ps1 " (%s)") $ '
```

Comment:
normal convention is to use @ instead of / i.e. `ubuntu@shared-gce-usw-01-prod:` / might get confused with the path right after it